keras/backend/tensorflow_backend.py

```diff
@@ -1866,6 +1866,26 @@ def batch_normalization(x, mean, var, beta, gamma, epsilon=1e-3):
     # Returns
         A tensor.
     """
+    if ndim(x) == 4:
+        # The CPU implementation of FusedBatchNorm only support NHWC
+        if len(mean.shape) == 1:
+            tf_data_format = 'NHWC'
+        else:
+            tf_data_format = None
+        if (tf_data_format is not None and
+                beta is not None and gamma is not None):
```

Comment:
If beta/gamma are None, we should create `ones` vectors for them so we can use fused ops anyway.

Reply:
Agree to extend.  
 If beta is None, let's to be zeros.
 If gamma is None, let's to be ones.